### PR TITLE
web: update the page metadata with the resource bar summary

### PR DESCRIPTION
**Pull request by nicks**
Hello @hyu,

Please review the following commits I made in branch nicks/ch11430:

76856df84d1af630d9f17f875cc9bdbc3f8962f9 (2021-02-19 12:55:09 -0500)
web: update the page metadata with the resource bar summary
Fixes https://github.com/tilt-dev/tilt/issues/4216

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
Screenshot from 2021-02-19 12-55-55

small quality of life thing that i've wanted a lot!

File: web/src/OverviewResourceBar.tsx [hunk 1]
```diff
@@ -231,6 +231,28 @@ function statusCounts(resources: Proto.webviewResource[]): StatusCounts {
...
+    let faviconHref = ""
+    if (unhealthy > 0) {
+      document.title = `Tilt | ❌ (${unhealthy})`
+      faviconHref = "/static/ico/favicon-red.ico"
+    } else if (pending || total === 0) {
+      document.title = `Tilt | ⏲  (${healthy}/${total})`
+      faviconHref = "/static/ico/favicon-gray.ico"
+    } else {
+      document.title = `Tilt | ✅ (${healthy}/${total})`
+      faviconHref = "/static/ico/favicon-green.ico"
```
Comment (hyu) [inline web/src/OverviewResourceBar.tsx:247]: What do you think about this?
- "Tilt" is the most boring bit of info, so we relegate it to the end (which gets truncated if you have a lot of tabs)
- The color of the Favicon is already informative, so the status icon should be grayscale (lest it clash)
- Nix the parens so the important numbers have a better chance of showing up when there are tons of tabs

```suggestion
      document.title = `✖︎ ${unhealthy} ┊ Tilt`
      faviconHref = "/static/ico/favicon-red.ico"
    } else if (pending || total === 0) {
      document.title = `… ${healthy}/${total} ┊ Tilt`
      faviconHref = "/static/ico/favicon-gray.ico"
    } else {
      document.title = `✔︎ ${healthy}/${total} ┊ Tilt`
      faviconHref = "/static/ico/favicon-green.ico"
```

**Review by hyu — APPROVED** [review]
Finally! Love this enhancement. 🤩

What do you think about using the Tilt "T" as our favicon? We've been moving away from the Windmill logo for the app itself, in line with our general divestment from the Windmill brand in favor of Tilt all the way. This might be mildly surprising to people who are already used to the existing favicon, but I think this PR is a good occasion for the change since we're offering a new feature. 

Updated Favicons for 3 statuses: [favicon-T.zip](https://github.com/tilt-dev/tilt/files/6013027/favicon-T.zip)

(I think we can keep the Windmill circle for our blog + docs, which will help with disambiguation if users have docs open alongside Tilt).

File: web/src/OverviewResourceBar.tsx (hunk 1 above)
Comment (nicks) [reply web/src/OverviewResourceBar.tsx:247]: love it!

**Comment by nicks** [comment]
great ideas! updated

**Pull request merged**